### DISCUSSION TITLE
Update manual installation instructions to be useful

### DISCUSSION THREAD

**Discussion by cwik-tech** [shadcn-ui/ui#1297] [Ideas]
Just tried to install accordion (https://ui.shadcn.com/docs/components/accordion) in my nextjs app and it simply does not work how the instruction says. The reason for that is because I did not have the imported import { cn } from '@lib/utils'. so after googling 2h I finally found the solution i.e. the 2 missing steps that are NOT mentioned (or intuitive) in the instructions: 1) install class-variance-authority 2) create the actual utils file. The full instructions are here: https://www.neorepo.com/blog/how-to-build-a-button-with-nextjs-and-shadcn-ui
For the creators of chadcn-ui, I know you are top-experts and these kind of things are very silly and obvious for you, but please write the instructions explicitly with step-by-step approach and run a "mum test" i.e. if your mum cannot understand it from just reading it, it means you did not do the explanation clearly enough.
Many thanks I love your work!

**Comment by igorm84** [comment]
@cwikio even manual installation suposes that you runned init cli ([see more here](https://ui.shadcn.com/docs/installation)) command that installs the dependencies, but maybe it should be more explicity , the problem is that major components use cn

**Reply by cwik-tech** [reply]
hi @igorm84 thanks for the clarification. I guess this is the solution as well. I tried running the init before but is completely wipes out my tailwind config (literately, deletes everything and replaces it with few items from chadcn-ui. I presume it should actually extend it, not replace it, but it is not the case). Thus, I reversed the entire init and decided to install it manually (redix + the instructions). if you have the admin rights, please a) make it explicit what you said b) add the full description of the manual way incl. installation of dependencies like I outlined above. Many thanks!
